### DISCUSSION TITLE
Getting `ERR_EMPTY_ASSERTION` when it hits the ACS endpoint

### DISCUSSION THREAD

**Discussion by bhagya1101** [tngan/samlify#526] [Q&A]
I am getting the above error whenever I am hitting my ACS endpoint. What do I need to do for it?
Any help will be appreciated.

**Comment by bhagya1101** [comment]
Is that I have an issue in my certificates or so?

**Comment by KuSh** [comment]
I've encountered this problem if assertion is encrypted in SamlHttpRequest but samlify IdentityProvider is not explicitly configured with `isAssertionEncrypted: true` (defaults to false)
